app/build.gradle.kts

```diff
@@ -208,6 +208,35 @@ android {
         // Debug
         debugImplementation("androidx.compose.ui:ui-tooling")
         debugImplementation("androidx.compose.ui:ui-test-manifest")
+
+        // Kapt dependencies
+        kapt("com.google.dagger:hilt-android-compiler:2.51")
+        kapt("com.google.dagger:hilt-compiler:2.51")
+        kaptAndroidTest("com.google.dagger:hilt-compiler:2.51")
+        kaptTest("com.google.dagger:hilt-compiler:2.51")
+        // Mockito
+        androidTestImplementation("org.mockito:mockito-core:5.11.0")
+
+        testImplementation("io.mockk:mockk:1.13.10")
+        androidTestImplementation("io.mockk:mockk:1.13.10")
+        androidTestImplementation("io.mockk:mockk-android:1.13.10")
```

Comment:
it seems like some of these already exist in the file above?

Reply:
you're right although they were already duplicates previous to me adding them

Reply:
Gotcha, eitherway its best to remove adding them more to prevent clutter